src/components/ProjectDialog.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, Dialog, Grid, IconButton, Typography, Link } from '@mui/material';
+import { Box, Dialog, Grid, IconButton, Typography, Link as MuiLink } from '@mui/material';
```

Comment:
Is there a reason for renaming `Link` to `MuiLink`? Since your still use Box, Dialog, Grid etc. I would leave `Link` as it is to keep it consistent, if there is no clash with another `Link` somewhere.